File: Modul 2/soal_4/PRAK204-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
banyak_variable = 2
inputs = []

while len(inputs) < banyak_variable:
    line = input()

    new_line = line.split()

    for value in new_line:
        inputs.append(float(value))

jariJari, tinggi = inputs
volume = 22/7 * (jariJari**2) * tinggi
luas = 2*22/7*jariJari*(jariJari+tinggi)
keliling = 2*22/7*jariJari

print("Volume = %.2lf\nLuas = %.2lf\nKeliling = %.2lf"%(volume,luas,keliling))

File: Modul 2/soal_4/PRAK204-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>
#define PI 22.0/7.0

int main(){
    float jariJari = 0;
    float tinggi = 0;

    scanf("%f %f", &jariJari, &tinggi);
    float volume = PI * (jariJari*jariJari) * tinggi;
    float luas = 2*PI*jariJari*(jariJari+tinggi);
    float keliling = 2*PI*jariJari;

    printf("Volume = %.2f\nLuas = %.2f\nKeliling = %.2f",volume,luas,keliling);
    return 0;
}